app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Event.java

```diff
@@ -89,4 +91,6 @@ public EventLocation getLocation() {
     public String getTwitterName() {
         return this.twitterName;
     }
+
+    // TODO put setters ??
```

Comment:
What'd be the setters for ?

Reply:
For event organisers to later modify the Event but this canc come later.